sphinxcontrib/chapeldomain/__init__.py

```diff
@@ -29,8 +29,9 @@
 from sphinx.util.docfields import Field, GroupedField, TypedField
 from sphinx.util.nodes import make_refnode
 
+from sphinxcontrib.chapeldomain.chapel import ChapelLexer
 
-VERSION = '0.0.18'
+VERSION = '0.0.19'
```

Comment:
The version needs to be updated in a separate PR when we've pushed a new release with this in it.  The latest release is 0.0.17, this one should go into 0.0.18